### ISSUE TITLE
Gradient message cannot set (fe) #0000FF

### ISSUE THREAD

**Issue by eeeepish** [Koenkk/zigbee2mqtt#16866]
### What happened?

When sending a message to a Phillips Lightstrip that supports the relatively recently added 'gradient' message, there are several colours that fail to set, most notably any that include '#0000FF' (so, for example, a message of '{"gradient":["#FFFFFF","#0000FF"]}' will fail).

I can see from logs that the message is sent to the MQTT broker correctly, and that Zigbee2MQTT receives the message, but with this colour (and a few others, mostly minor variants) it does not then respond with an (example) message of 'MQTT publish: topic 'zigbee2mqtt/Gradient Bias', payload '{"brightness":253,"color":{"x":0.6942,"y":0.2963},"color_mode":"xy","color_temp":158,"gradient":["#ffffff","#ff0500"],"linkquality":83,"power_on_behavior":null,"state":"ON","update":{"installed_version":16782080,"latest_version":16782080,"state":"idle"},"update_available":null}'' (such a message is never sent when using the gradient specified, and no errors are logged, and for the record this is what is returned when sending the gradient ["FFFFFF", "FF000000"]...)

(I'm using a Hue Play gradient lightstrip 55 - model [929002422702](https://www.zigbee2mqtt.io/devices/929002422702.html#philips-929002422702))

### What did you expect to happen?

When sending a gradient message to a Hue light which supports the feature, which includes the colour '#0000FF', the light will change to a gradient including this colour, rather than failing silently

### How to reproduce it (minimal and precise)

Send a message to a device supporting the recently added Gradient endpoint for Hue devices which includes the colour of '#0000FF' (e.g. {"gradient":["#FFFFFF","#0000FF"]})

### Zigbee2MQTT version

1.30.2

### Adapter firmware version

20221226

### Adapter

Sonoff

### Debug log

<img width="1692" alt="Screenshot 2023-03-01 at 20 16 37" src="https://user-images.githubusercontent.com/126716747/222256094-b0a2bacf-b97c-40d9-967d-4bc9176fe48a.png">

(apologies, my Synology docker container doesn't appear to be saving logs in a way that I can upload them. I'm only technologically literate to the point I'm dangerous, so I'm very happy to provide a full log but someone may have to ELI5 for me to be able to do it...)

**Comment by macmanroy** [comment]
Same problem here
